src/components/TopNavigationBar/TopNavigationBar.tsx

```diff
@@ -113,36 +136,130 @@ export default function TopNavigationBar({
                 >
                   Problems
                 </Link>
-                {userGroups.data?.length > 0 && (
-                  <Link
-                    to="/groups/"
-                    getProps={({ isCurrent }) => ({
-                      className: isCurrent
-                        ? 'inline-flex items-center px-1 pt-0.5 border-b-2 border-blue-500 dark:border-blue-700 text-base font-medium leading-6 text-gray-900 dark:text-dark-high-emphasis focus:outline-none focus:border-blue-700 dark:focus:border-blue-500 transition'
-                        : 'inline-flex items-center px-1 pt-0.5 border-b-2 border-transparent text-base font-medium leading-6 text-gray-500 hover:text-gray-700 hover:border-gray-300 dark:text-dark-high-emphasis focus:outline-none focus:text-gray-700 focus:border-gray-300 transition',
-                    })}
-                  >
-                    Groups
-                  </Link>
-                )}
-                <a
-                  href="https://forum.usaco.guide/"
-                  target="_blank"
-                  rel="noreferrer"
-                  className="inline-flex items-center px-1 pt-0.5 border-b-2 border-transparent text-base font-medium leading-6 text-gray-500 hover:text-gray-700 hover:border-gray-300 dark:text-dark-high-emphasis focus:outline-none focus:text-gray-700 focus:border-gray-300 transition"
-                >
-                  Forum
-                </a>
-                <a
-                  href="https://ide.usaco.guide/"
-                  target="_blank"
-                  rel="noreferrer"
-                  className="inline-flex items-center px-1 pt-0.5 border-b-2 border-transparent text-base font-medium leading-6 text-gray-500 hover:text-gray-700 hover:border-gray-300 dark:text-dark-high-emphasis focus:outline-none focus:text-gray-700 focus:border-gray-300 transition"
+                <Popover.Group
+                  as="nav"
+                  className="group inline-flex items-center border-b-2 border-transparent hover:border-gray-300 focus-within:border-gray-300 transition"
                 >
-                  IDE
-                </a>
+                  <Popover>
+                    {({ open }) => (
+                      <>
+                        <Popover.Button
+                          className={classNames(
+                            open ? 'text-gray-900' : 'text-gray-500',
+                            'group inline-flex items-center space-x-2 text-base leading-6 font-medium group-hover:text-gray-900 focus:outline-none focus:text-gray-900 transition ease-in-out duration-150 dark:text-dark-high-emphasis'
```

Comment:
Oh yeah I could try that. I used group hover because if you hover over the general area but not specifically on the "Sections" or "Resources" text, the border will show up but the text color doesn't darken (for light mode)

Reply:
Another fix would be to make the buttons fill the height of the nav bar, but idk how to do that when they are nested in another div

Reply:
Okay yay I was able to fix everything, can you see if I there are other changes needed after it deploys?